back/db/migrate/20220822140950_add_fields_and_rename_field_in_static_pages.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+class AddFieldsAndRenameFieldInStaticPages < ActiveRecord::Migration[6.1]
+  # rubocop:disable Rails/ApplicationRecord
+  class StubStaticPage < ActiveRecord::Base
+    self.table_name = 'static_pages'
+  end
+  # rubocop:enable Rails/ApplicationRecord
+
+  def change
+    rename_column :static_pages, :body_multiloc, :top_info_section_multiloc
+    change_column_null :static_pages, :top_info_section_multiloc, false
+    add_column :static_pages, :banner_enabled, :boolean, default: true, null: false
```

Comment:
It will be `true` for all new policy pages. Is it expected?

Reply:
I would say this is undecided atm. As you mentioned elsewhere (can't remember where), this can be set at create so it doesn't really matter. We can also easily change this if we feel it is better with a default of `false`.

Reply:
But I see what you mean. I would need to change all the policy page seeds to include a `false` setting for this field, and probably also in the internal templates. Maybe it's easier to change its default value to `false`, and let the FE set it to `true` as desired.

Reply:
I've set the default to `false` plus run `db:migrate:up` + `db:migrate:down` to rebuild schema, etc.